client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-type/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { Onboard } from '@automattic/data-stores';
+import { StepContainer } from '@automattic/onboarding';
+import { __experimentalVStack as VStack } from '@wordpress/components';
```

Comment:
Could use `Flex` component instead to avoid using experimental component.

Reply:
It works as is, so I'm tempted to leave it be.

Reply:
Just double check it works with package update: https://github.com/Automattic/wp-calypso/pull/78711

I noticed some changes to VStack/Hstack in that PR.